packages/apollo-codegen-swift/src/__tests__/__snapshots__/codeGeneration.ts.snap

```diff
@@ -1069,7 +1069,7 @@ exports[`Swift code generation #structDeclarationForFragment() should generate a
   public var friends: [Friend?]? {
     get {
       return (resultMap[\\"friends\\"] as? [ResultMap?]).flatMap { (value: [ResultMap?]) -> [Friend?] in value.map { (value: ResultMap?) -> Friend? in value.flatMap { (value: ResultMap) -> Friend in Friend(unsafeResultMap: value) } } }?.filter({
-        guard let type = $0 else { return false}; return Friend.possibleTypes.contains($type.__typename)
+        guard let type = $0 else { return false }; return Friend.possibleTypes.contains($type.__typename)
```

Comment:
Seeing this I think you're not aware of how these snapshots are usually updated. 

If you run `npm test -- -u`, it will automatically write the new snapshots (after which you should of course check if they are correct), so you never have to write them by hand.

Reply:
Unfortunately for reasons I don't totally understand, `npm test` is totally failing locally. Otherwise yes, that's what I should be doing